system/settings/settings.xml

```diff
@@ -2253,7 +2235,7 @@
         </setting>
         <setting id="audiooutput.ac3transcode" type="boolean" parent="audiooutput.ac3passthrough" label="667" help="36429">
           <level>2</level>
-          <default>false</default>
+          <default>true</default>
           <dependencies>
```

Comment:
Yes, AC3 transcode is too heavy for Pi1.
It is usable on Pi2/Pi3. Not sure about it being enabled by default.